app/decorators/hash_diff_decorator/base.rb

```diff
@@ -19,13 +19,21 @@ def to_html
     def diff_val(val)
       if val.nil?
         "[nil]"
+      elsif val.is_a?(Numeric)
+        diff_numeric(val)
       elsif val.empty?
         "[empty]"
-      elsif val.is_a?(Numeric)
-        format('%.2f', val)
       else
         val.inspect
       end
     end
+
+    def diff_numeric(val)
+      if val.is_a?(Fixnum)
+        val.to_s
+      elsif val.is_a?(Numeric)
```

Comment:
what this be a float? If so, what about just checking for that? (just looking up `Numeric` in Ruby docs since I don't see this class used too often)